src/io/file_io.cpp

```diff
@@ -47,6 +47,8 @@ struct LocalFile : VirtualFileReader, VirtualFileWriter {
   }
 
   size_t Write(const void* buffer, size_t bytes) const {
+    // 4 bytes alignment
+    CHECK_EQ(bytes % 4, 0);
```

Comment:
it looks like this check is failing for many of the CI jobs (and not just R)

![image](https://user-images.githubusercontent.com/7608904/94515165-96dc6a00-01e8-11eb-8f88-7f630eab6400.png)